app/controllers/application-overview.ts

```diff
@@ -11,14 +11,24 @@ function getApplicationOverview(req: Request, res: Response, next: NextFunction)
       name: loggedInUserFullName,
       applicationNextStep: getAppealApplicationNextStep(req),
       history: getAppealApplicationHistory(),
-      stages
-
+      stages,
+      saved: req.session.appeal.application.isPartiallySaved
     });
```

Comment:
Talking with Alex about the isPartiallySaved we don't think this should be saved in the session but just added as a query parameter.